core/src/main/java/org/apache/iceberg/actions/BaseRewriteDataFilesAction.java

```diff
@@ -277,6 +279,15 @@ private void replaceDataFiles(Iterable<DataFile> deletedDataFiles, Iterable<Data
     }
   }
 
+  private boolean isSingleBigFile(Map.Entry<StructLikeWrapper, Collection<FileScanTask>> kv) {
```

Comment:
This should be fine just taking a single FileScanTask as an arg right? Since we only call this method on a single FileScan task to see if it is splittable?

I would probably also say 
isSplittable(FileScanTask file)

Reply:
Yes,I udpate it